samples/test/test.js

```diff
@@ -15,14 +15,16 @@
 
 'use strict';
 
-const execa = require('execa');
+const {spawn} = require('child_process');
+const path = require('path')
 
 describe('debug samples', () => {
 
   it('should run the quickstart', done => {
     // select a random port between 49152 and 65535
     const PORT = Math.floor((Math.random() * (65535-49152))) + 49152;
-    const proc = execa('node', ['app.js'], {
+    const proc = spawn('node', ['app.js'], {
+      shell: true,
       env: {
```

Comment:
I think the problem here is that `execa` merged environment by default, which is different than the default behavior.

```
env: Object.assign(process.env, {PORT})
```

👆 I think that would fix things.

Reply:
Good find!